README.md

```diff
@@ -298,6 +298,12 @@ Delete physical extension
 DELETE /physicalextensions/{extension|mac}
 ```
 
+Change admin user web interface password for all physical extensions
+
+```
+POST /physicalextensions/adminpw/{adminpw}
```

Comment:
As adminpw is a secret I'd really like to not show it in the request URL...

Can we put it in the payload intead?